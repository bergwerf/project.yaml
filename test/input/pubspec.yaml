name: newtify
version: 1.2.3
homepage: http://newtify.dartlang.org
documentation: http://docs.newtify.com
dependencies:
  efts: ^2.0.4
  transmogrify: ^0.4.0
dev_dependencies:
  test: '>=0.6.0 <0.12.0'
dependency_overrides:
  transmogrify:
    path: ../transmogrify_patch/
