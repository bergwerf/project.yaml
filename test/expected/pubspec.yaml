name: 'My Project'
version: 1.0.0
homepage: 'http://example.com'
documentation: 'http://docs.newtify.com'
dependencies:
efts: ^2.0.4
transmogrify: ^0.4.0
dev_dependencies:
test: '>=0.6.0 <0.12.0'
dependency_overrides:
transmogrify: {path: ../transmogrify_patch/}
authors:
- 'John Doe'
- 'John Doe'
- 'Joe Bloggs'
- 'Joe Bloggs <joe@bloggs.com>'
- 'Joe Bloggs <joe@bloggs.com> (http://joebloggs.com)'
description: 'Awesome project!'

